docs/end-user-flows/sign-up-and-sign-in/README.mdx

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_position: 1
+---
+
+# Sign-up and sign-in
+
+Sign-in and sign-up is the core interaction process for end-users to authenticate and authorize access to client applications. As a centralized OIDC-based CIAM platform, Logto provides a universal sign-in experience for users across multiple client applications and platforms.
```

Comment:
The title says `sign-up and sign-in` but the main content starts with `sign-in and signup`. Reads a bit odd.

Reply:
updated